packages/js/product-editor/src/components/iframe-editor/header-toolbar/header-toolbar.tsx

```diff
@@ -144,9 +144,13 @@ export function HeaderToolbar( {
 							icon={ plus }
 							label={
 								! isInserterOpened
-									? __( 'Add', 'woocommerce' )
+									? __(
+											'Toggle block inserter',
+											'woocommerce'
+									  )
 									: __( 'Close', 'woocommerce' )
```

Comment:
I think we just want to always show `Toggle block inserter`. Currently it will show `Close` if the panel is open.

Reply:
Done in commit [5e71d25](https://github.com/woocommerce/woocommerce/pull/50232/commits/5e71d255fe7e92962c3ba031e9cbcec565d07482)